backbone.nativeajax.js

```diff
@@ -11,6 +11,13 @@
   } else if (typeof exports === 'object') { module.exports = factory();
   } else { Backbone.ajax = factory(); }
 }(function() {
+
+  if (!Array.isArray) {
```

Comment:
Converted to a local function.  Do you still prefer adding note to docs over an extra couple lines of code?